### ISSUE TITLE
[documentation] Incorrect setup instructions for Homebrew-installed pyenv.

### ISSUE THREAD

**Issue by coredumperror** [pyenv/pyenv#2000]
The new(ish?) instructions output by running `pyenv init` assume that pyenv is installed in `~/.pyenv`. But when you install via Homebrew, which the README specifically tells you to do if you're on MacOS, it's actually installed in `/usr/local/var/pyenv`

This incongruity just cost me a ton of time trying to figure out what was even wrong with what I'd done, as my call to `eval "$(pyenv init -)"` in .bashrc kept complaining about how it no longer sets up `PATH` for you, and none of my shims were in my `PATH`. 

It turns out that what I needed to do was change 
```export PYENV_ROOT="$HOME/.pyenv"```
To 
```export PYENV_ROOT="/usr/local/var/pyenv"```

Perhaps `pyenv init` could be updated to intelligently tell you where pyenv is actually installed, to avoid this in the future?

**Comment by native-api** [comment]
> it's actually installed in `/usr/local/var/pyenv`

What makes you think so?

```
admin@Mac-Admin ~ % brew list pyenv
/usr/local/Cellar/pyenv/2.0.2/bin/pyenv
/usr/local/Cellar/pyenv/2.0.2/bin/pyenv-install
/usr/local/Cellar/pyenv/2.0.2/bin/pyenv-uninstall
/usr/local/Cellar/pyenv/2.0.2/bin/python-build
/usr/local/Cellar/pyenv/2.0.2/COMMANDS.md
/usr/local/Cellar/pyenv/2.0.2/completions/ (3 files)
/usr/local/Cellar/pyenv/2.0.2/CONDUCT.md
/usr/local/Cellar/pyenv/2.0.2/Dockerfile
/usr/local/Cellar/pyenv/2.0.2/libexec/ (26 files)
/usr/local/Cellar/pyenv/2.0.2/Makefile
/usr/local/Cellar/pyenv/2.0.2/plugins/python-build/ (672 files)
/usr/local/Cellar/pyenv/2.0.2/plugins/.gitignore
/usr/local/Cellar/pyenv/2.0.2/pyenv.1
/usr/local/Cellar/pyenv/2.0.2/pyenv.d/exec/ (4 files)
/usr/local/Cellar/pyenv/2.0.2/pyenv.d/install/latest.bash
/usr/local/Cellar/pyenv/2.0.2/pyenv.d/rehash/ (6 files)
/usr/local/Cellar/pyenv/2.0.2/src/ (7 files)
/usr/local/Cellar/pyenv/2.0.2/terminal_output.png
/usr/local/Cellar/pyenv/2.0.2/test/libexec/pyenv-echo
/usr/local/Cellar/pyenv/2.0.2/test/ (26 files)
```
```

**Comment by coredumperror** [comment]
```
$ pyenv root
/usr/local/var/pyenv
```

That's how my system is set up. It's been way too long since I installed pyenv to remember if I did anything unusual with the install, but I doubt it.

**Comment by jimmywan** [comment]
@coredumperror I just updated pyenv to 2.0.3 via brew and I have no files in /usr/local/var/pyenv.
My install also broke, but not for the reason you stated. Perhaps you have 2 versions of pyenv installed?
`$ pyenv root`

**Comment by coredumperror** [comment]
Hmmm. I think my problem stems from an env var that's been sitting my in .bashrc for who knows how long...

```export PYENV_ROOT=/usr/local/var/pyenv
```

Why is that there? I have a very vague recollection of setting that up, but I can't recall why I chose that folder... Maybe a much older version of Pyenv, perhaps installed via MacPorts rather than Homerew (I switched several years ago), had that root, and I didn't want to reinstall my envs when I switched to Homebrew?

Either way, it's now clear why I was having a problem, and that it wasn't Pyenv's fault. So I'll close this issue.
